source/compiler/aslrestype2d.c

```diff
@@ -158,11 +158,168 @@
 /*
  * This module contains the Dword (32-bit) address space descriptors:
  *
+ * DWordPcc
  * DwordIO
  * DwordMemory
  * DwordSpace
  */
 
+
+/*******************************************************************************
+ *
+ * FUNCTION:    RsDoDwordPccDescriptor
+ *
+ * PARAMETERS:  Info                - Parse Op and resource template offset
+ *
+ * RETURN:      Completed resource node
+ *
+ * DESCRIPTION: Construct a long "DwordIO" descriptor
```

Comment:
I think this should say DwordPcc instead of DwordIO

Reply:
Done

Reply:
Thanks for the review Rob!